### ISSUE TITLE
Giving no links

### ISSUE THREAD

**Issue by olaban-ji** [Anshuman-Verma/chia-anime-mass-downloader#19]
I ran this code in the command propmt `C:\Users\HP\Desktop\chia-anime-downloader-master>python chia_anime_downloader.py -D http://www.chia-anime.tv/episode/fate-zero/ 1-25 1080p -v -l`
Then I get this
![debug](https://user-images.githubusercontent.com/22367812/34859092-7bdac7f2-f754-11e7-8c8c-a5fa1e8b6aad.png)
Then when the script has finished running I get a fate-zero folder and a fate-zero .txt file which are both empty but it's strange because it's meant to store the links to the .txt file
![fate zero](https://user-images.githubusercontent.com/22367812/34859137-bb66fc92-f754-11e7-87dc-0270631b9bbf.png)
From the image below you'll notice that after selecting the option of saving link it justs exits
![debug2](https://user-images.githubusercontent.com/22367812/34859291-92370ffa-f755-11e7-9a25-a14b34b40924.png)

**Comment by anshumanv** [comment]
Hey @rawdpiper thanks for reporting this, I think it's some issue with the website, can you please check if you're able to view episodes online, eg - http://www.chia-anime.tv/fate-zero-episode-1-english-subbed/
:smile: 

I'm getting a white screen when I play it online.

**Comment by olaban-ji** [comment]
@anshumanv tried playing the video on my end and it worked both chia-anime server and anime premium server worked. Did the issue that occured to me also occur on your end?

**Comment by anshumanv** [comment]
Not sure what's the problem then, at my end the episode isn't loading - 
![image](https://user-images.githubusercontent.com/21009455/34902927-19f8a84e-f84c-11e7-835a-dcd1c4b87c4c.png)

@rawdpiper

**Comment by Yureien** [comment]
@anshumanv I'm having some trouble on chia anime too. Want to make a scrapper for masterani.me ? The website has a lot more anime collection.

**Comment by olaban-ji** [comment]
@anshumanv, when I wanted to setup the script, I ran the setup.py and it got point the setup requested I get Microsoft visual C++ for one of the requirements and then I decided to install the requirements individually using pip install and they all installed, so I'm thinking maybe it's because of the Microsoft visual c++ that I didn't install that's why the script is not able to write the .txt file properly

@fadedcoder that would be great if you guys can do that for masterani.me

**Comment by anshumanv** [comment]
@FadedCoder I'm in for it! :smile: 

I think it's Vue framework, you think we can make this work @FadedCoder? There's no separate search page too.

**Comment by Yureien** [comment]
Yep, I made a masteranime scraper a few months back. We just need to do some minor fixes.

**Comment by Yureien** [comment]
https://github.com/jQwotos/anime_scrapers/blob/master/scrapers/masteranime.py

Check this. Masterani.me has a hidden API.

**Comment by Yureien** [comment]
I'll create a repo soon. That code prob won't work. I have to modify it.

**Comment by olaban-ji** [comment]
@anshumanv Is Microsoft Visual C++ needed for the installation of the script?

**Comment by Yureien** [comment]
Nope only python.

**Comment by olaban-ji** [comment]
> Nope only python.

So any idea as to why the script's not saving the link to the .txt file? Are you also facing the same issue?

**Comment by Yureien** [comment]
I'm guessing it's because it can't find the links. Will run the script soon
and find the exact reason. I'm building the master anime scraper now,
actually.

**Comment by Yureien** [comment]
Instead of scraping anime websites, want to scrape a anime torrenting site
like nyaa.si? That will be more useful as it has the biggest collection.

**Comment by anshumanv** [comment]
Yes, that can be done @FadedCoder, but the motive of these batch downloaders is to actually be able to grasp batch download links so they can be downloaded in download managers like idm, uget etc Batch anime torrents are anyways available. :smile:

**Comment by Yureien** [comment]
Alright then. We will build a Master Anime one.

**Comment by Itsune9tl** [comment]
It's 2019 and for what ever reason trying to watch videos from Chia-anime.tv while using Firefox produces a white screen again. No Error code. just a blank white screen. just to check I used chrome, and the Videos play just fine. Did they put an exclusion in the web code?
